lib/taskana-core/src/test/java/acceptance/workbasket/update/UpdateWorkbasketAuthorizationsAccTest.java

```diff
@@ -173,6 +173,32 @@ void testUpdatedAccessItemLeadsToNotAuthorizedException() throws Exception {
     assertThatThrownBy(call).isInstanceOf(NotAuthorizedToQueryWorkbasketException.class);
   }
 
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_setReadTasksPerm() throws Exception {
+    final WorkbasketService workbasketService = taskanaEngine.getWorkbasketService();
+    String wbId = "WBI:100000000000000000000000000000000006";
+
+    List<WorkbasketAccessItem> accessItems = workbasketService.getWorkbasketAccessItems(wbId);
+    WorkbasketAccessItem theAccessItem =
+        accessItems.stream()
+            .filter(x -> "user-1-1".equalsIgnoreCase(x.getAccessId()))
+            .findFirst()
+            .orElse(null);
+    assertThat(theAccessItem).isNotNull();
+    theAccessItem.setPermission(WorkbasketPermission.READTASKS, false);
+    workbasketService.updateWorkbasketAccessItem(theAccessItem);
+
+    List<WorkbasketAccessItem> accessItems2 = workbasketService.getWorkbasketAccessItems(wbId);
```

Comment:
@gitgoodjhe after updating Permission Workbasket Access Item, should the Access Item be queried again to check if Permission is updated? In other tests, access items are queried again, which actually is not needed.